module/core/src/Infrastructure/Service/Decorator/CacheDownloaderDecorator.php

```diff
@@ -23,14 +23,14 @@ public function __construct(DownloaderInterface $downloader, KernelInterface $ap
         $this->directory = sprintf('%s/var/downloader', $appKernel->getProjectDir());
     }
 
-    public function download(string $url, array $headers = []): string
+    public function download(string $url, array $headers = [], string $acceptedHeaderTypes = null): string
```

Comment:
As far as I know it is about limiting the allowed mime formats ? So maybe it is better to call it AcceptedMimeTypes or AcceptetContenType ?